salt/transport/tcp.py

```diff
@@ -553,6 +736,10 @@ def __init__(
         source_ip=None,
         source_port=None,
     ):
+        salt.utils.versions.warn_until(
```

Comment:
dunder deprecation mentioned earlier

Reply:
Dunder deprecate only works on things imported by the loader. It won't work here.

Reply:
Thanks for letting me know, didn't know that.
Disregard all my dunder observations then